Storage 'local' not found in /etc/pve/storage.cfg

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

homelable

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/homelable.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

pve-manager/9.1.6/71482d1833ded40a (running kernel: 6.17.13-2-pve)

### 📝 Provide a clear and concise description of the issue.

Hello,

I have the log below.
Impossible to install because of the local storage not found while I have as you can see in the logs.

✖️ Storage 'local' not found in /etc/pve/storage.cfg
✖️ in line 534: exit code 213 (Proxmox: Storage type does not support 'rootdir' content): while executing command exit 213

Proxmox :

Version : 9.6.1
Standalone
Debian

Using the script with default option and advanced, it is the same

### 🔄 Steps to reproduce the issue.

execute the commande in the console

### ❌ Paste the full error output (if available).

/ / / /___ ____ ___ ___ / /___ / / / /__
/ // / __ / __ __ \/ _ \/ / __ / __ / / _
/ __ / // / / / / / / / / // / // / / /
// //_// // /_/_//_,/.//__/

🧩 Using Advanced Install on node proxme

💡 PVE Version 9.1.6 (Kernel: 6.17.13-2-pve)
🖥️ Operating System: debian
🌟 Version: 13
📦 Container Type: Unprivileged
🆔 Container ID: 120
🏠 Hostname: homelable
💾 Disk Size: 8 GB
🧠 CPU Cores: 2
🛠️ RAM Size: 2048 MiB
🌉 Bridge: vmbr1
📡 IPv4: dhcp
📡 IPv6: disable
🗂️ FUSE Support: no
📦 Nesting: Enabled
📦 Keyctl: Enabled
🎮 GPU Passthrough: no
💡 Timezone: Europe/Paris
🔍 Verbose Mode: yes
🚀 Creating an LXC of Homelable using the above advanced settings
✔️ Updated app defaults: /usr/local/community-scripts/defaults/homelable.vars
✔️ Storage local (Free: 2.5TB Used: 1.2TB) [Template]
✔️ Storage local (Free: 2.5TB Used: 1.2TB) [Container]
⏳ Validating storage 'local'grep: /etc/pve/storage.cfg: No such file or directory
✖️ Storage 'local' not found in /etc/pve/storage.cfg
✖️ in line 534: exit code 213 (Proxmox: Storage type does not support 'rootdir' content): while executing command exit 213

--- Last 20 lines of log ---
[2026-04-20 13:08:53] Container ID: 120
[2026-04-20 13:08:53] Hostname: homelable
[2026-04-20 13:08:53] Disk Size: 8 GB
[2026-04-20 13:08:53] CPU Cores: 2
[2026-04-20 13:08:53] RAM Size: 2048 MiB
[2026-04-20 13:08:53] Bridge: vmbr1
[2026-04-20 13:08:53] IPv4: dhcp
[2026-04-20 13:08:53] IPv6: disable
[2026-04-20 13:08:53] FUSE Support: no
[2026-04-20 13:08:53] Nesting: Enabled
[2026-04-20 13:08:53] Mknod: Disabled
[2026-04-20 13:08:53] GPU Passthrough: no
[2026-04-20 13:08:53] Verbose Mode: yes
[2026-04-20 13:08:53] Session ID: db1734fd
[2026-04-20 13:09:11] [OK] Updated app defaults: /usr/local/community-scripts/defaults/homelable.vars
[2026-04-20 13:09:16] [OK] Storage local (Free: 2.5TB Used: 1.2TB) [Template]
[2026-04-20 13:09:16] [OK] Storage local (Free: 2.5TB Used: 1.2TB) [Container]
[2026-04-20 13:09:16] [INFO] Validating storage 'local'
[2026-04-20 13:09:16] [ERROR] Storage 'local' not found in /etc/pve/storage.cfg
[2026-04-20 13:09:18] [ERROR] in line 534: exit code 213 (Proxmox: Storage type does not support 'rootdir' content): while executing command exit 213


### 🖼️ Additional context (optional).

_No response_

--- a/misc/build.func
+++ b/misc/build.func
@@ -5678,6 +5678,10 @@ create_lxc_container() {
 
   if ! pvesm status -content rootdir 2>/dev/null | awk 'NR>1{print $1}' | grep -qx "$CONTAINER_STORAGE"; then
     msg_error "Storage '$CONTAINER_STORAGE' ($STORAGE_TYPE) does not support 'rootdir' content."
+    msg_custom "💡" "${YW}" "Enable 'Disk image' (rootdir) for storage '${CONTAINER_STORAGE}' in:"
+    msg_custom "   " "${YW}" "Datacenter → Storage → ${CONTAINER_STORAGE} → Edit → Content"
+    msg_custom "📖" "${YW}" "See: https://pve.proxmox.com/wiki/Storage"
+    msg_custom "🔗" "${YW}" "Help: https://github.com/community-scripts/ProxmoxVE/discussions"
     exit 213
   fi
   msg_ok "Storage '$CONTAINER_STORAGE' ($STORAGE_TYPE) validated"

